### DISCUSSION TITLE
Different colours on different windows

### DISCUSSION THREAD

**Discussion by longj0n** [dbeaver/dbeaver#16141] [Q&A]
Using DBeaver 22.0.1 on Win 10, I tend to open up a few different windows with all tabs in 1 window for a specific DB connection, useful to help prevent doing stuff on the wrong DB. But what would be even more useful is a way to colour-code each window. Doesn't need to be an automatic colour-allocation based on DB connection as that would need to be per tab, but come to think of it that might be useful as well...? But even just a toolbar button to set background or windows header bar colour for that window, but that window alone, would be great. Not sure if this is even possible in Windows though, unless using completely customised "skin" type system, a la Winamp and similar.

Thanks all for maintaining this superb and essential tool.

**Comment by LonwoLonwo** [comment]
Hello @longj0n 

Maybe ability to create custom connections type with the coloring can help you?

You find it in the Connection settings -> General -> Edit connection types.

Usually, our users use Production connection type to mark important databases. 
And you can create your own connection type.

**Comment by LonwoLonwo** [comment]
![2022-04-07 16_28_07-DBeaver Enterprise 21 3 3 - Playlist](https://user-images.githubusercontent.com/45152336/162210778-ec189793-5569-4c93-a8c4-284a90a75b4c.png)

**Comment by longj0n** [comment]
Hi Anastasiya, thank you so much, yes this is of course exactly what I  
need to prevent nasty accidents!

I should have known this was already a feature but didn't realise  
where to find it.

Kind regards,

Jon


Quoting Anastasiya ***@***.***>:

**Comment by LonwoLonwo** [comment]
Thanks for the answer @longj0n
